.ci/install_cri_containerd.sh

```diff
@@ -23,12 +23,13 @@ source "${script_dir}/lib.sh"
 #Use cri contaienrd tarball format.
 #https://github.com/containerd/cri/blob/master/docs/installation.md#release-tarball
 CONTAINERD_OS=$(go env GOOS)
-CONTAIENRD_ARCH=$(go env GOARCH)
+CONTAINERD_ARCH=$(go env GOARCH)
 
 cri_containerd_tarball_version=$(get_version "externals.cri-containerd.version")
 cri_containerd_repo=$(get_version "externals.cri-containerd.url")
 
 cri_containerd_version=${cri_containerd_tarball_version#v}
+cri_containerd_pr="5911" # TODO - put in version.yaml and pull dynamically $(get_version "externals.cri-containerd.pr_id")
```

Comment:
This looks like a temporary tweak which can be dropped when it gets merged by the upstream. If so, it might be safer to add a `do-not-merge` label to this PR. That way, we can still test if the code works in the Kata CI, but we won't land it by mistake.

Alternatively, if you preceive it may take a while to get that PR landed, are you happy to update this script once the PR does land?

Either way, I think it would be useful to add a call like the following to make this stand out in the CI output:

```bash
warn "Using patched Confidential Computing containerd version: see https://github.com/containerd/containerd/pull/5911"
```

(Note that it's best to spell out what "CC" refers to as for some of the "old hands", that acronym can also stand for https://github.com/clearcontainers :smile:)

Reply:
Hey @jodh-intel, thanks very much for the review and suggestions. I've added the warn statement, but am a little unsure about the hard-coded version change. It's a bit of chicken and egg and I originally made the proper change in version.yaml, but this required an update in `kata-containers` repo that I couldn't test with this PR, so I was going to get this in, then update kata-containers, then come back to tests and fix, but that's not necessarily very efficient.
Alternatively I do have a PR open in that repo now (https://github.com/kata-containers/kata-containers/pull/2556), so I could put the version change in there, and make this change proper and depend on that?

Reply:
As an update I decided to make the version change in https://github.com/kata-containers/kata-containers/pull/2556, so once that has been merged I'll update the code here to remove the TODO'd temporary code in this PR

Reply:
As a further update, based on feedback from Samuel, I've separated the version change from the PR and created #2577 as a stand-alone PR with this change in now.